app/commands/v2/discard_draft.rb

```diff
@@ -47,7 +47,7 @@ def delete_draft_from_draft_content_store(draft_path)
       def send_live_to_draft_content_store(live)
         PresentedContentStoreWorker.perform_async(
           content_store: Adapters::DraftContentStore,
-          payload: Presenters::ContentStorePresenter.present(live, event),
+          payload: Presenters::ContentStorePresenter.present(live, event, [:draft, :published]),
```

Comment:
I'm not sure I follow this, does it send both the live and the published item?  Is this the fallback order?